src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax;
+
+public class FigureSupplier {
+    private static final int FIGURES_AMOUNT = 5;
+
+    private static int getFigureNumber() {
+        return (int) (1 + Math.random() * FIGURES_AMOUNT);
+    }
+
+    private static int getRandomNumber() {
+        return (int) (1 + Math.random() * 100);
+    }
```

Comment:
it's bad practice to use static methods, lets make them not static. Also we can create one method getRandomNumber(int maxValue); that accepts max value of random number, and pass numbers like FIGURES_AMOUNT as parameter